deepchem/models/wandblogger.py

```diff
@@ -17,9 +17,28 @@ class WandbLogger(object):
     will log the specified metrics calculated on the specific datasets
     to the user's W&B dashboard.
 
-    If a WandbLogger is provided to the wandb_logger flag,
+    If a WandbLogger is provided to the wandb_logger flag of the model,
```

Comment:
The argument needs to be renamed to just `logger`.

Reply:
I thought renaming the argument to `logger` was part of moving it into the standardized logging interface (which is still in development). Did you want me to do the renaming now for this current version? I can submit another PR just to rename this flag for KerasModel/TorchModel?

Reply:
Done

Reply:
Let's just finish up the new logger API before merging this.

Reply:
@peastman any updates on this? Would be nice to get this merged since the docs would be useful for the standardized logging API too.

Reply:
Let's get the new logger API finished first.

Reply:
Ok, should we also take out the old existing `wandb` flag for the new logging API while we're at it?

Reply:
No, we need to support it for backward compatibility.  It should be marked as deprecated though.